.github/workflows/release-please.yml

```diff
@@ -31,3 +31,24 @@ jobs:
             console.log('Releases Created:', '${{ steps.release.outputs.releases_created }}');
             console.log('Paths Released:', '${{ steps.release.outputs.paths_released }}');
             console.log('PRs Created:', '${{ steps.release.outputs.prs }}');
+      - name: Install yq
```

Comment:
I think you don't have to install explicitly, just add `uses: mikefarah/yq@master` like in their example:

```
  - name: Install yq
        uses: mikefarah/yq@master
```

Then yq should be available?
```
- name: Set foobar to cool
    uses: mikefarah/yq@master
    with:
      cmd: yq -i '.foo.bar = "cool"' 'config.yml'
  - name: Get an entry with a variable that might contain dots or spaces
    id: get_username
    uses: mikefarah/yq@master
    with:
      cmd: yq '.all.children.["${{ matrix.ip_address }}"].username' ops/inventories/production.yml
```